pkg/gofr/gofr_test.go

```diff
@@ -504,3 +504,107 @@ func Test_AddCronJob_Success(t *testing.T) {
 
 	assert.Truef(t, pass, "unable to add cron job to cron table")
 }
+
+func TestStaticHandler(t *testing.T) {
+	// Generate a public directory endpoint
+	directory := "./public"
+
+	if _, err := os.Stat(directory); err != nil {
+		err := os.Mkdir("./public", 0755)
```

Comment:
Use "public" variable here and everywhere

Reply:
sure